dev/ProjectReunion_DLL/GetRawNotificationEventArgs.h

```diff
@@ -0,0 +1,34 @@
+﻿
+#include <pch.h>
+#include "PushNotificationReceivedEventArgs.h"
+#include "externs.h"
+
+using namespace winrt;
+using namespace winrt::Windows::Foundation;
+
+constexpr PCWSTR reunionPushServerArg = L"-ReunionPushServer";
+
+namespace GetRawNotificationEventArgs {
+    static Windows::Foundation::IInspectable GetRawNotificationEventArgs() {
+        Windows::Foundation::IInspectable args = nullptr;
+
+        std::wstring commandLineArgs = GetCommandLine();
+        auto found = commandLineArgs.find(reunionPushServerArg, 0);
+        if (found != std::wstring::npos)
+        {
+            {
+                auto lock = g_lock.lock();
+                THROW_HR_IF_NULL_MSG(E_UNEXPECTED, g_waitHandleForArgs, "PushNotificationManager::RegisterActivator has not been called.");
+            }
+
+            DWORD waitResult = WaitForSingleObject(g_waitHandleForArgs.get(), 1000);
```

Comment:
Nit: Can eliminate local waitResult by directly calling WaitForSWingleObject in the if ( ... ) clause